hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1379,6 +1379,29 @@
       milestone: V2
       tests: []
     }
+    {
+      name: chip_sram_execution
+      desc: '''
+            Enable instruction execution from both the main and retention SRAMs.
+            Have the CPU fetch instruction from both and verify that the SW test can still finish
+            correctly.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: chip_sram_lc_escalation
+      desc: '''
+            Trigger an escalation request to the SRAMs from the LC_CTRL, and check that the SRAM
+            stops accepting and responding to new memory requests.
+            Reset the system to exit the terminal escalated state, re-initialize the SRAMs, and
+            verify that they can now respond correctly to any further memory requests.
+
+            X-ref'ed with LC_CTRL tests for any common methods.
```

Comment:
Since we are testing the effect of escalation on SRAM (i.e. specific to SRAM), we don't need to mention the X-reference.

Reply:
Done, thanks